### ISSUE TITLE
DNSBL source "justdomains.github.io" is too outdated. we need different sources

### ISSUE THREAD

**Issue by OoSTARTERoO** [opnsense/core#6610]
- **Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Is your feature request related to a problem? Please describe.**

This request is more like a report of a problem.
on "Unbound DNS" of OpnSense, there are predefined sources of blocklist
many of the sources depend on [justdomains](https://justdomains.github.io/)

But, it hasn't been updated since **3 Oct 2022**, and seems to be abandoned
[justdomains/ci/issues/7](https://github.com/justdomains/ci/issues/7)

**Describe the solution you like**
Changing the source of those lists would be great.
possible source could be [firebog](https://firebog.net/)

But better way of executing this would be [taking the source from the justdomains repo as it is on MIT License](https://github.com/justdomains/ci/blob/master/convertlists.py) and converting the list on the local

I am really sorry if I didn't follow the common guideline of opening an issue on github.
this is my first time opening a issue, so if i did anything wrong, please tell me :)

**Comment by OoSTARTERoO** [comment]
upon further testing, code still works if correct link is fed.
tested with List-KR, adaway, Easylist

Adguard DNS file needs some modification on line 564 as it seems to be cp949.
adding encoding='UTF8' solves this issue

**Comment by OoSTARTERoO** [comment]
I was able to run and update the blocklist locally.

1. make a cron job to run a Python script on a daily basis
2. link nginx to point the text file (ex http://192.168.0.1/adguard.txt)
3. add it to URLs of Blocklists

Uploading the file to OpnSense's nginx plugin was possible, but it simply rejects the request of updating the URLs of Blocklists.
it seems to be somewhat related to the custom configuration of the plugin that forcefully adds some configuration (like blocking some http user agent). As the OpnSense itself kept being banned from nginx.

So, I had to upload it to a different server I had, and it is working!
it takes some time, but if anyone finds this hope this helps
